tests/Dfe.PlanTech.Web.UnitTests/Background/BackgroundTaskHostedServiceTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Dfe.PlanTech.Application.Background;
+using Dfe.PlanTech.Web.Background;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+
+namespace Dfe.PlanTech.Web.UnitTests.Background;
+
+public class BackgroundTaskHostedServiceTests
+{
+    [Fact]
+    public async Task ExecuteAsync_Runs_One_WorkItem_And_Logs()
+    {
+        // Arrange
+        var logger = Substitute.For<ILogger<BackgroundTaskHostedService>>();
+        var queue = Substitute.For<IBackgroundTaskQueue>();
+
+        var ran = false;
+        var cts = new CancellationTokenSource();
+
+        // Dequeue once, then cancel so the loop exits
+        queue.DequeueAsync(Arg.Any<CancellationToken>())
+             .Returns(Task.FromResult<Func<CancellationToken, Task>>(ct =>
+             {
+                 ran = true;
+                 cts.Cancel(); // stop the loop after this item
+                 return Task.CompletedTask;
+             }));
+
+        var sut = new BackgroundTaskHostedService(logger, queue);
+
+        // Act
+        await sut.StartAsync(cts.Token); // StartAsync calls ExecuteAsync under the hood
+
+        // Assert
+        Assert.True(ran);
+        await queue.Received(1).DequeueAsync(Arg.Any<CancellationToken>());
+
+        // At least two info logs: starting + read item
+        logger.ReceivedWithAnyArgs().Log(LogLevel.Information, 0, default!, null, default!);
```

Comment:
The comment and the assertion don't seem to align.

i.e. "at least two info logs" versus "we had one or more info level logs"